### ISSUE TITLE
Unhelpful comment when the chem equ editor is used

### ISSUE THREAD

**Issue by rh584** [ucam-cl-dtg/isaac-app#852]
If you type in the wrong equation in the Chem equ editor, using it for a nuclear equ, then if anything is incorrect you get the abrupt comment

incorrect
Your answer contains invalid syntax!
Please try again.

It is just that the answer is not the correct one though. The syntax may be fine but the answer is not correct. This makes it seem like there is something wrong with the approach rather than one of the values is incorrect.

**Comment by Morpheu5** [comment]
Good catch. What phrasing would you rather see?

**Comment by jsharkey13** [comment]
It actually is invalid syntax, from the point of view of the parser in the checker; it fails to parse the `\betaparticle` I think? It would be worth seeing what question this was on, but I saw this go by in the logs yesterday:

```
==================================================
Syntax error
Syntax error
gcse_ch6_52_q4|5ef70778-145e-4b19-8af1-cf6e2e295ced
--------------------------------------------------
Target string: '{}^{90}_{38}Sr  -> ^{0}_{-1}\betaparticle +  {}^{90}_{39}Y'
Test string: '{}^{90}_{38}Sr  -> \betaparticle +  {}^{90}_{39}Y'
Parse Expressions
instead expected token classes are [error, LPAREN, LSQUARE, NUMBER, SUP, SUB, ELEMENT, ELECTRON, FRACTION]
instead expected token classes are [error, LPAREN, LSQUARE, NUMBER, SUP, SUB, ELEMENT, ELECTRON, FRACTION]
Parsed target: 'ERROR'
Parsed test: 'ERROR'
ERROR: Trusted string contains error!
        "{}^{90}_{38}Sr  -> ^{0}_{-1}\betaparticle +  {}^{90}_{39}Y"
        "ERROR"
Equality:
        Equal: false
        Weakly equivalent: false
        Contains Error: true
==================================================
```

and the "Contains Error" bit is the bit that leads to that response . . .

**Comment by rh584** [comment]
Yes, 52.4 is one typical question on this issue of the beta. There are several representations of the beta particle.
^{0}_{-1}\betaparticle  used here and in question were you count up the numbers on the subscript and superscript.
\betaparticle used generally in textbooks,
\betaparticle^-1 more widely used in textbooks,
In the questions, you have to do the counting, but it is not clear that you have to write the over-embellished form of the beta wit the numbers attached. It is not stated anywhere. This is rather for beginners. But that is who we have, so we may have to accept all three varieties of beta. This requires a "consultation". As does the unhelpful wording.
Another issue was raised in conjunction with this, which is that if the order of the produced particles is interchanged, the answer is marked incorrect. This may not be a different issue; as has been raised.

**Comment by Morpheu5** [comment]
OK, so this has less to do with the equation editor and more with the checker, right?
